src/FunctionApps/python/GenerateCSVs/fhir.py

```diff
@@ -12,17 +14,49 @@
 RECORD_TYPE_ECR = "ecr"
 
 
-def read_bundles_by_type() -> Iterator[Tuple[str, dict]]:
+def get_blobs(
```

Comment:
$0.02: I've been debating whether or not to make an `azure` package at the top level to deal with some of this - any thoughts there?

Reply:
I think it's a good idea.  A lot of the code was copied and tweaked from IntakePipeline and could be standardized and pulled into a central place.